src/Workspaces/SharedUtilitiesAndExtensions/Workspace/VisualBasic/Extensions/ContextQuery/VisualBasicSyntaxContext.vb

```diff
@@ -147,6 +154,47 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions.ContextQuery
             Return False
         End Function
 
+        Private Shared Function ComputeIsBaseClassContext(targetToken As SyntaxToken) As Boolean
+            If targetToken.IsChildToken(Of InheritsStatementSyntax)(Function(n) n.InheritsKeyword) AndAlso targetToken.HasAncestor(Of ClassBlockSyntax)() Then
+                Return True
+            End If
+
+            Return targetToken.IsChildToken(Of QualifiedNameSyntax)(Function(n) n.DotToken) AndAlso
+                   targetToken.Parent?.FirstAncestorOrSelf(Of InheritsStatementSyntax) IsNot Nothing AndAlso
+                   targetToken.HasAncestor(Of ClassBlockSyntax)()
```

Comment:
stil missing something.  if i have `implements IList(of X.$$)` it looks like ComputeIsBaseClassContext will return 'true' here.